models/prices/prices_tokens.sql

```diff
@@ -1756,5 +1757,56 @@ VALUES
     ("usdt-tether", "solana", "USDT", "Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB", 6),
     ("drip-drip-network", "bnb", "DRIP", "0x20f663CEa80FaCE82ACDFA3aAE6862d246cE0333", 18),
     ("fur-furio", "bnb", "$FUR", "0x48378891d6E459ca9a56B88b406E8F4eAB2e39bF", 18),
-    ("pstn-piston", "bnb", "PSTN", "0xBfACD29427fF376FF3BC22dfFB29866277cA5Fb4", 18)
+    ("pstn-piston", "bnb", "PSTN", "0xBfACD29427fF376FF3BC22dfFB29866277cA5Fb4", 18),
+
+    ("weth-weth", "fantom", "WETH", "0x74b23882a30290451A17c44f4F05243b6b58C76d", 18), 
+    ("wbtc-wrapped-bitcoin", "fantom", "WBTC", "0x321162Cd933E2Be498Cd2267a90534A804051b11", 8),
+    ("ftm-fantom", "fantom", "WFTM", "0x21be370D5312f44cB42ce377BC9b8a0cEF1A4C83", 18),
+    ("aave-new", "fantom", "AAVE", "0x6a07A792ab2965C72a5B8088d3a069A7aC3a993B", 18), 
+    ("avax-avalanche", "fantom", "AVAX", "0x511D35c52a3C244E7b8bd92c0C297755FbD89212", 18), 
+    ("crv-curve-dao-token", "fantom", "CRV", "0x1E4F97b9f9F913c46F1632781732927B9019C68b", 18),
+    ("dai-dai", "fantom", "DAI", "0x8D11eC38a3EB5E956B052f67Da8Bdc9bef8Abf3E", 18),
+    ("frax-frax", "fantom", "FRAX", "0xdc301622e621166BD8E82f2cA0A26c13Ad0BE355", 18),
+    ("renbtc-renbtc", "fantom", "renBTC", "0xDBf31dF14B66535aF65AaC99C32e9eA844e14501", 8),
+    ("usdc-usd-coin", "fantom", "USDC", "0x04068DA6C83AFCFA0e13ba15A6696662335D5B75", 6), 
+    ("tusd-trueusd", "fantom", "TUSD", "0x9879aBDea01a879644185341F7aF7d8343556B7a", 18), 
+    ("fxs-frax-share", "fantom", "FXS", "0x7d016eec9c25232b01F23EF992D98ca97fc2AF5a", 18),
+    ("mcrt-magiccraft", "fantom", "MCRT", "0xE705aF5f63fcaBDCDF5016aA838EAaac35D12890", 9), 
+    ("tarot-tarot", "fantom", "TAROT", "0xC5e2B037D30a390e62180970B3aa4E91868764cD", 18),
+    ("brush-brush", "fantom", "BRUSH", "0x85dec8c4B2680793661bCA91a8F129607571863d", 18), 
```

Comment:
`ERROR:root:WARN: brush-brush Provided contract address: 0x85dec8c4B2680793661bCA91a8F129607571863d not in CoinPaprika contracts list ['api response missing contracts field']. (Not uncommon! share block explorer link to confirm contract)`
- API: https://api.coinpaprika.com/v1/coins/brush-brush
- SCAN: https://ftmscan.com/token/0x85dec8c4B2680793661bCA91a8F129607571863d